lib/src/main/java/uk/gov/di/ipv/cri/passport/library/error/ErrorResponse.java

```diff
@@ -15,7 +15,9 @@
     ERROR_GETTING_RESPONSE_FROM_DCS(1006, "No response was returned from DCS"),
     DCS_RETURNED_AN_ERROR(1007, "DCS returned an error response"),
     MISSING_SHARED_ATTRIBUTES_JWT(1008, "Missing shared attributes JWT from request body"),
-    FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT(1009, "Failed to parse shared attributes JWT");
+    FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT(1009, "Failed to parse shared attributes JWT"),
+    MISSING_CLIENT_QUERY_PARAMETERS(1010, "Missing query parameters for client id"),
```

Comment:
Could you rename this variable and slightly change the error message description to something like 

MISSING_CLIENT_ID_QUERY_PARAMETER(1010, "Missing client_id query parameter")

I think something like this would be a bit clearer that we are specifying just one very specific query parameter that is missing?

Reply:
Refactor this